### PR TITLE
RK-6946 - Adding open-tracing doc file.

### PR DESCRIPTION
**Pull request by noagoldman**
This is a draft of the open tracing documentation.
Please feel free to add comments about what else we should add / if you think we should add links to other sites explaining about open tracing.

### CODE REVIEW

File: docs/open-tracing.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+---
+id: open-tracing
+title: OpenTracing
+sidebar_label: OpenTracing
+---
+
+OpenTracing is an open standard for distributed tracing, which allows to trace requests from beginning to end across multiple touchpoints and microservices
```
Comment (LiranLast) [inline docs/open-tracing.md:7]: Add a link :)

File: docs/open-tracing.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+in a large stage applications.  
+
+Common use cases:
+* Microservices - follow the journey transactions takes through microservices architecture.
+* Caching — troubleshooting to determine whether a request is hitting the cache. 
+* Arbitration — tracing the full history of a single process,
+determining its behavior when multiple services contact it in parallel rather than sequentially.
+
+Different tools implements OpenTracing standard by providing a timeline view of requests using the OpenTracing terminology (spans, tags, span context etc...).
+Rookout can integrate with such tools and fetch tracing data.
```
Comment (LiranLast) [inline docs/open-tracing.md:17]: Not enough information on what we integrate with, how to set it up, etc... :)

File: website/sidebars.json [hunk 3]
```diff
@@ -29,7 +29,8 @@
       "collaborations",
       "agent-setup",
       "breakpoints-structure",
-      "breakpoints-reference"
+      "breakpoints-reference",
```
Comment (LiranLast) [inline website/sidebars.json:32]: This should probably be higher up

File: docs/open-tracing.md (hunk 2 above)
Comment (noagoldman) [reply docs/open-tracing.md:17]: in my opinion, we should wait for users to connect with us and only then explain how to integrate it. I think we haven't tried it yet with a real customer and we don't want someone to try and implement it without discussing it with us, only to find out that it (maybe) won't work. 
I think we want to keep the buzz but still not share enough so users will talk to us first about it. 
Only my opinion, you call :)

File: docs/open-tracing.md (hunk 1 above)
Comment (noagoldman) [reply docs/open-tracing.md:7]: To the openTracing site? Thought about it, wasn't sure, I will add it

File: docs/open-tracing.md (hunk 2 above)
Comment (LiranLast) [reply docs/open-tracing.md:17]: This reads more like a marketing doc :)

File: docs/open-tracing.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
+---
+id: open-tracing
+title: OpenTracing
+sidebar_label: OpenTracing
+---
+
+OpenTracing is an open standard for distributed tracing, which allows to trace requests from beginning to end across multiple touchpoints and microservices
```
Comment (Rookeret73) [inline docs/open-tracing.md:7]: which allows to trace requests -> which allows tracing of requests
large stage applications -> large scale applications

File: docs/open-tracing.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+in a large stage applications.  
+
+Common use cases:
+* Microservices - follow the journey transactions takes through microservices architecture.
```
Comment (Rookeret73) [inline docs/open-tracing.md:11]: follow the journey a transaction takes

File: docs/open-tracing.md [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+determining its behavior when multiple services contact it in parallel rather than sequentially.
+
+Different tools implements OpenTracing standard by providing a timeline view of requests using the OpenTracing terminology (spans, tags, span context etc...).
+Rookout can integrate with such tools and fetch tracing data. 
+
+Click here to learn more about [OpenTracing](https://opentracing.io/)
+
+**Fetch tracing data**
+1. Set up a Breakpoint. 
+2. Make sure Tracing checkbox is checked - go to BP editor, under collection check the Tracing checkbox.
```
Comment (Rookeret73) [inline docs/open-tracing.md:23]: Make sure the Tracing checkbox is checked in the the Breakpoint Editor -> Collection section.
Trigger the Breakpoint //try and be consistent. Breakpoint or BP work, not bp

File: docs/open-tracing.md [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+Different tools implements OpenTracing standard by providing a timeline view of requests using the OpenTracing terminology (spans, tags, span context etc...).
+Rookout can integrate with such tools and fetch tracing data. 
+
+Click here to learn more about [OpenTracing](https://opentracing.io/)
+
+**Fetch tracing data**
+1. Set up a Breakpoint. 
+2. Make sure Tracing checkbox is checked - go to BP editor, under collection check the Tracing checkbox. 
+3. Trigger the bp and open a message
+4. On the right side of the message pane, click on the tracing tab.
```
Comment (Rookeret73) [inline docs/open-tracing.md:25]: Tracing (capital T). Again, consistancy

File: docs/open-tracing.md [hunk 8]
```diff
@@ -0,0 +1,29 @@
...
+5. Now you will be able to see all the tracing data.
```
Comment (Rookeret73) [inline docs/open-tracing.md:26]: step 5 is redundant

File: docs/open-tracing.md [hunk 9]
```diff
@@ -0,0 +1,29 @@
...
+
+Rookout Tracing information is available for Java and Python applications that implement OpenTracing.\
```
Comment (Rookeret73) [inline docs/open-tracing.md:28]: unnecessary \ at the end of the line

**Review by BenTheKlein — APPROVED** [review]

**Pull request merged**